wsagent/che-core-api-project/src/test/java/org/eclipse/che/api/project/server/ProjectServiceTest.java

```diff
@@ -928,6 +939,50 @@ public void testDeleteProject() throws Exception {
     }
 
     @Test
+    public void testDeleteProjectsConcurrently() throws Exception {
+        int threadNumber = 100;
+        ExecutorService executor = Executors.newFixedThreadPool(threadNumber);
```

Comment:
Well you're right, though in this very case it should not necessarily be shut down except for the reason of reclamation of its resources, but I guess I will add a call to make the code more clear.